src/openvisuspy/slice.py

```diff
@@ -855,25 +938,24 @@ def setSceneBody(self, scene):
 			directions = {it: I for I, it in enumerate(directions)} if directions else  {'X':0,'Y':1,'Z':2}
 		self.direction.options=directions
 
-		self.timestep_delta.value=int(scene.get("timestep-delta", 1))
+		self.timestep_delta.value=int(scene.get("timestep-delta", 64))
 		self.timestep.value=int(scene.get("timestep", self.db.getTimesteps()[0]))
 		self.view_dependent.value = bool(scene.get('view-dependent', True))
 
 		resolution=int(scene.get("resolution", -6))
 		if resolution<0: resolution=self.db.getMaxResolution()+resolution
 		self.resolution.end = self.db.getMaxResolution()
 		self.resolution.value = resolution
-
 		self.field.value=scene.get("field", self.db.getField().name)
-		self.num_refinements.value=int(scene.get("num-refinements", 1 if pdim==1 else 2))
+		self.num_refinements.value=int(scene.get("num-refinements", 1))

```

Comment:
I am scared by interactivity setting `1` by default. At least with 2 you have a way to get the  first coarse level and move/abort query quickly...